spec/spec_helper.rb

```diff
@@ -27,6 +28,7 @@
   config.before(:suite) do
     DatabaseCleaner.strategy = :transaction
     DatabaseCleaner.clean_with(:truncation)
+    TestAfterCommit.enabled = true
```

Comment:
So, I read through the gem README but wasn't *entirely* sure ... what does `test_after_commit` do for us?